source/plugin/text/representations/xml.rst

```diff
@@ -0,0 +1,51 @@
+===============
+TextXML Format
+===============
+
+Sponge uses a proprietary form of `XML <https://en.wikipedia.org/wiki/XML>`_ called "TextXML" to denote the coloring
```

Comment:
proprietary is a bit strong of a word, considering it wasn't sponges to begin with but adapted with permission from @TomyLobo

The differences that I am aware of  between TextXML and XML is the schema is already defined, you don't need a wrapping root node for a valid fragment(Sponge adds this automatically), and whitespace is counted.